docs/aqp_aws.md

```diff
@@ -1,550 +1,235 @@
+
+##Deploying the Cluster with AWS CloudFormation##
+SnappyData uses the Amazon EC2 with AWS Cloud Formation method to automatically install, configure and start a SnappyData cluster. In this release, the configuration supports launching the cluster on a single EC2 instance. 
+This method is recommended as the quickest way to deploy SnappyData. All you need is an existing AWS account and login credentials to get started! 
+###Configuring and Launching the Cluster###
+The configure and launch the cluster:
+1. Go to http://www.snappydata.io/cloudbuilder. 
+On this page, you need to complete the steps to configure and start the cluster.
+2. Enter the name for your cluster. Each clusters is identified by it’s unique name. The names and details of the members are automatically derived from the provided cluster name.
+
+3. Enter a name of an existing EC2 KeyPair. This enables SSH access to the cluster. Please refer to the Amazon documentation for more information on generating your own key pair.	
+ 
+4.  Select an instance and storage based on the capacity that you need..
+ 
+5. Currently only Amazon Elastic Block Storage (EBS) is supported by SnappyData.
+
+6. Enter a value (between 256GB - 1024 GB) in the EBS Volume Size(gigabytes) field.
+
+7. Click Generate. 
+8. On the next page, select the AWS region, and then click Launch Cluster. 
+Note: Ensure that the key pair is created in the region selected.
+ 
+9. The Select Template page allows you to select a template that describes the stack you want to create. By default, the URL for the Amazon S3 template is provided. Click Next. 
+	
+10. Enter a stack name or continue to use the default value. Click Next.
+11. Specify the tags  or leave the field empty to use the default values. Click Next.
+12. On the Review page, verify the details and click Create to create a stack. 
+	Note: This operation may take a few minutes to complete.  
+13. The next page lists the existing stacks. Click Refresh to update the list and and the current status of the stack. Ensure that the status of the stack is “Create_Complete”. 
+14. Open a web browser and enter the http://<Public DNS>:<Port Number> to launch Apache Zeppelin. You can find the public DNS by loading the stack’s EC2 home page.
+For more information on using Notebooks, refer to Using Predefined Notebooks.	
+##Deploying the Cluster on AWS using Scripts##
+SnappyData provides a script that allows you to launch and manage SnappyData clusters on Amazon Elastic Compute Cloud (EC2). 
+Download the script from the [SnappyData Release page.](https://github.com/SnappyDataInc/snappydata/releases/) . The package is available in compressed files (**snappy-ec2.tar**). Extract the contents to a location on your computer.
+Using the snappy**-ec2** script, you can identify each of the clusters by it's unique cluster name. The script internally ties members (locators, leads and stores) of the cluster with EC2 security groups. 
```

Comment:
I did not really understand this line "_Using the *snappy-ec2 *script, you can identify each of the clusters by it's unique cluster name._"